apps/src/code-studio/pd/application_dashboard/detail_view_contents.jsx

```diff
@@ -181,17 +183,17 @@ export class DetailViewContents extends React.Component {
   };
 
   handleRegionalPartnerChange = (selected) => {
-    const regional_partner_filter = selected ? selected.value : UnmatchedFilter;
+    const regional_partner_value = selected ? selected.value : UnmatchedValue;
     const regional_partner_name = selected ? selected.label : UnmatchedLabel;
-    this.setState({ regional_partner_name, regional_partner_filter});
+    this.setState({ regional_partner_name, regional_partner_value});
   };
 
   handleSaveClick = () => {
     let stateValues = [
       'status',
       'locked',
       'notes',
-      'regional_partner_filter',
+      'regional_partner_value',
```

Comment:
Since this is the instance of the dropdown used to assign an application to a regional partner (editing the app in detail view), it doesn't use Redux because it shouldn't update the filter for the application dashboard.